src/zenml/integrations/huggingface/materializers/huggingface_tf_model_materializer.py

```diff
@@ -46,30 +45,30 @@ def load(self, data_type: Type[TFPreTrainedModel]) -> TFPreTrainedModel:
         Returns:
             The model read from the specified dir.
         """
-        temp_dir = TemporaryDirectory()
-        io_utils.copy_dir(
-            os.path.join(self.uri, DEFAULT_TF_MODEL_DIR), temp_dir.name
-        )
-
-        config = AutoConfig.from_pretrained(temp_dir.name)
-        architecture = "TF" + config.architectures[0]
-        model_cls = getattr(
-            importlib.import_module("transformers"), architecture
-        )
-        return model_cls.from_pretrained(temp_dir.name)
+        with self.get_temporary_directory(delete_at_exit=False) as temp_dir:
+            io_utils.copy_dir(
+                os.path.join(self.uri, DEFAULT_TF_MODEL_DIR), temp_dir
```

Comment:
I saw this pattern in some other materializers as well. Does the removal of `.name` lead to any different behavior or is it just the same?